lib/response.js

```diff
@@ -133,8 +134,6 @@ module.exports = {
     const original = this._body;
     this._body = val;
 
-    if (this.res.headersSent) return;
```

Comment:
we shouldn't return here because there are some specific logic for streams below.